targets/targets.json

```diff
@@ -1193,6 +1193,7 @@
         }
     },
     "MTB_USI_WM_BN_BM_22": {
+        "overrides": {"lse_available": 0},
```

Comment:
Thanks @kivaisan ..
1. I think this will also be the case for MTB_ADV_WISE_1530 as well. Can you please check the schematics for 1530 and move this to the parent target if suitable? 
2. Minor nit on coding standards.. can you please follow coding conventions for 1 statement per line?